src/router/index.ts

```diff
@@ -81,6 +76,11 @@ const routes: Array<RouteConfig> = [
         name: 'Payment',
         component: () => import('../views/Payment.vue'),
     },
+    {
+        path: '/claims',
+        name: 'Claim',
+        component: () => import('../views/Claim.vue'),
```

Comment:
@AlmarMG Take the payment route as a start for your metamask work and do not remove the original claim route since it is used for token claims.

Move this back into the router config:
```
{
        path: '/claim',
        name: 'Claim',
        component: () => import('../views/Claim.vue'),
},
```
Create `src/views/Claims.vue` based on `src/views/Payment.vue` and add this route:
```
{
        path: '/claims',
        name: 'Claims',
        component: () => import('../views/Claims.vue'),
}
```